.env

```diff
@@ -5,8 +5,10 @@ PUBLIC_URL=${UI_DEPLOY_PATH_PREFIX}/apps/subscription-reporting/
 
 REACT_APP_AJAX_TIMEOUT=60000
 
+REACT_APP_CONFIG_SERVICE_LOCALES_COOKIE=rh_locale
 REACT_APP_CONFIG_SERVICE_LOCALES_DEFAULT_LNG=en
```

Comment:
request: 
- this'll change to `en-US` along with a couple of other locations...
- `./public/locales/en.json` --> en-US.json
- `./public/locales/locales.json` --> the English references inside need updates

research:
we may be able to have the i18n plugin strip the locale subset for us with a config setting... meaning we wouldn't have to update anything going from `en` -> `en-US`
- https://www.i18next.com/
- https://react.i18next.com/